src/Features/VisualBasic/Portable/Wrapping/InitializerExpression/AbstractVisualBasicInitializerExpressionWrapper.vb

```diff
@@ -0,0 +1,21 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.VisualBasic.Indentation
+Imports Microsoft.CodeAnalysis.Wrapping.InitializerExpression
+
+Friend MustInherit Class AbstractVisualBasicInitializerExpressionWrapper(
```

Comment:
same question as before.  Doesn't seem like we need `AbstractVisualBasicInitializerExpressionWrapper` given there's only one derived type of it.  can you just merge them?

Reply:
Yea, that makes sense. I went ahead and merged them into one.